Userland/DevTools/Profiler/Process.cpp

```diff
@@ -70,10 +70,10 @@ static MappedObject* get_or_create_mapped_object(String const& path)
 void LibraryMetadata::handle_mmap(FlatPtr base, size_t size, String const& name)
 {
     StringView path;
+    if (!name.contains(':'))
+        return;
     if (name.contains("Loader.so"sv))
         path = "Loader.so"sv;
```

Comment:
I'm pretty sure this can be removed completely now, as we can now get the path to the Loader by just parsing the mapping name as usual. That also means that we can also make the following line look like the following and remove the `else` branch completely:

```
auto path = name.substring_view(0, name.view().find(':').value());
```

Also, the second commit deserves a new title and description then. :^)

Reply:
You're right. I've updated the PR